### ISSUE TITLE
.tld doesn't include .co.in and .co.kr (as well as) .co.ma

### ISSUE THREAD

**Issue by Procyon-b** [Tampermonkey/tampermonkey#1246]
## Actual Behavior
google.co.in and google.co.kr are not caught by this line:
`// @include https://*.google.tld/*`

14 other google.co.XX (not from major countries) have the same problem. I don't know if they must be fixed too.

## Specifications

- Chrome
- TM: 4.13
- OS: windows

## Script

www.google.co.uk
www.google.co.in
www.google.co.kr

```js
// ==UserScript==
// @name         test TLD
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  try to take over the world!
// @author       You
// @include      https://*.google.tld/*
// @grant        none
// @noframes
// ==/UserScript==

alert('matched');
```

**Comment by Procyon-b** [comment]
@derjanb are you interested in a list of the other 14 .co.xx tlds ?
These are domains that google deemed interesting enough to serve google search on it.

**Comment by Procyon-b** [comment]
Morocco's tld has also this problem: **.co.ma**
www.google.co.ma

**Comment by derjanb** [comment]
Should be fixed at 4.14.6141
